src/hypothesis/core.py

```diff
@@ -55,6 +62,11 @@
 from hypothesis.internal.conjecture.engine import ExitReason, \
     ConjectureRunner, uniform, sort_key
 
+try:
+    from coverage.tracer import CFileDisposition as FileDisposition
+except ImportError:  # pragma: no cover
```

Comment:
When would we hit this ImportError, and why can’t we test it?

Reply:
If we're on a platform without a C compiler where there is no binary wheel distributed for Coverage. Even if we did test it (which you're right, we currently don't), it still wouldn't get counted under coverage.

Reply:
Actually we do test it - under pypy we trigger that import error.